src/main/java/lotto/domain/LottoStore.java

```diff
@@ -3,24 +3,59 @@
 import java.util.ArrayList;
 import java.util.List;
 import lotto.settings.LottoSettings;
-import lotto.util.UniqueRandomNumberProvider;
 
 public class LottoStore {
 
     private static final Money BASE_AMOUNT = new Money(LottoSettings.DEFAULT_PRICE.value());
 
-    public List<Lotto> buy(Money money) {
+    private final LottoNumberGenerator lottoNumberGenerator;
+
+    public LottoStore(LottoNumberGenerator lottoNumberGenerator) {
+        this.lottoNumberGenerator = lottoNumberGenerator;
+    }
+
+    public Lottos buy(Money money, List<String> passivityLosttsInput) {
```

Comment:
```suggestion
    public Lottos autoLottos(Money money) {
```

현재 메서드는 많은 기능을 담당하고 있는 것은 아닐까?
위와 같이 Money를 전달하면 Money에 해당하는 금액 만큼의 로또를 자동으로 생성하는 역할만 두면 어떨까?